src/utils/ethereumUtils.ts

```diff
@@ -152,7 +152,7 @@ const getAsset = (
   return accountAssets[loweredUniqueId];
 };
 
-const getAccountAsset = (
+export const getAccountAsset = (
```

Comment:
this is already exported along with the other util functions at the bottom of the file